content/documentation/pages/3-stream-developer-guides/2-streams/1-standalone-stream-rabbitmq.md

```diff
@@ -233,79 +350,267 @@ Now let's build the Usage Cost Logger application. In the directory `usage-cost-
+To run the `UsageCostProcessor` application, you need to set the `input` binding to the test RabbitMQ exchange `test-usage-detail` to receive the `UsageDetail` data and the `output` binding to the test RabbitMQ exchange `test-usage-cost` to send the computed `UsageCostDetail`.
+You also need to set the `server.port` to access the actuator endpoints.
 
 ```
 spring.cloud.stream.bindings.input.destination=test-usage-detail
 spring.cloud.stream.bindings.output.destination=test-usage-cost
+server.port=9002
 ```
 
 You can run the standalone `UsageCostProcessor` processor application as,
 
 ```
-java -jar target/usage-cost-processor-rabbit-0.0.1-SNAPSHOT.jar --spring.cloud.stream.bindings.input.destination=test-usage-detail --spring.cloud.stream.bindings.output.destination=test-usage-cost &
+java -jar target/usage-cost-processor-rabbit-0.0.1-SNAPSHOT.jar --spring.cloud.stream.bindings.input.destination=test-usage-detail --spring.cloud.stream.bindings.output.destination=test-usage-cost --server.port=9002 &
+```
+
+Now, you can see that the `UsageCostProcessor` application receives the messages from the `RabbitMQ` exchange `test-usage-detail`, processes the cost and send the result to the `test-usage-cost` RabbitMQ exchange.
+
+From the RabbitMQ console:
+
+![Standalone Usage Cost Processor RabbitMQ](images/standalone-rabbitmq-usage-cost-processor.png)
+
+You can also see that there is an `anonymous` auto-delete queue created that consumes the messages from `test-usage-detail` exchange.
+This `anonymous` queue is bound to the `UsageCostProcessor` application's input.
+
+From the RabbitMQ console:
+
+![Standalone Usage Cost Processor RabbitMQ Anonymous Queue](images/standalone-rabbitmq-usage-cost-processor-anonymous-queue.png)
+
+If you want the processor application to consume from the `durable` queue, you need to make use of the `group` bindings property for the `UsageCostProcessor` application - provided you already have set up the `requiredGroups` for the `UsageDetailSender` application above.
+
+```
+spring.cloud.stream.bindings.input.group=usage-detail
+```
+
+Also, if you want the `UsageCostProcessor` application's output message delivery guarantee, then you need to set the `requiredGroups` for the `output` binding:
+
+```
+spring.cloud.stream.bindings.output.producer.requiredGroups=usage-cost
+```
+
+Now, you can run the `UsageCostProcessor` application as:
+
+```
+java -jar target/usage-cost-processor-rabbit-0.0.1-SNAPSHOT.jar --spring.cloud.stream.bindings.input.destination=test-usage-detail --spring.cloud.stream.bindings.output.destination=test-usage-cost --server.port=9002 --spring.cloud.stream.bindings.input.group=usage-detail --spring.cloud.stream.bindings.output.producer.requiredGroups=usage-cost &
```

Comment:
Perhaps it is better to remove `&` from the command. Let the user see the console output as opposed to running it in the background.